xml/bp_troubleshooting_pgs.xml

```diff
@@ -13,6 +13,46 @@
    <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
+ <sect1 xml:id="bp-troubleshooting-identify-pg">
+  <title>Identifying Troubled Placement Groups</title>
+  <para>
+   As previously noted, a placement group is not necessarily problematic
+   because its state is not 'active+clean'. Generally, &ceph;’s ability to
+   self repair may not be working when placement groups get stuck. The stuck
+   states include:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     <emphasis role="bold">Unclean</emphasis>: Placement groups contain
+     objects that are not replicated the required number of times. They should
+     be recovering.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis role="bold">Inactive</emphasis>: Placement groups cannot
+     process reads or writes because they are waiting for an OSD with the most
+     up-to-date data to come back up.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis role="bold">Stale</emphasis>: Placement groups are in an
+     unknown state, because the OSDs that host them have not reported to
+     MONs in a while (configured by the <option>mon osd report
+     timeout</option> option).
+    </para>
+   </listitem>
+  </itemizedlist>
```

Comment:
Just a suggestion -- might a <variablelist> be better for a list of states and their meanings?

Reply:
Probably :) I'll leave this for now, but when I got through the guide completely for edits, I'll change it up!

Reply:
OK!